extensions/wikia/Wall/WallController.class.php

```diff
@@ -49,7 +49,7 @@ public function messageDeleted() {
 	 *
 	 * @desc Renders old User_talk:[username] page in new place, using Wall_renderOldUserTalkPage.php template
 	 *
-	 * @author Andrzej 'nAndy' √Ö¬Åukaszewski
+	 * @author Andrzej 'nAndy' Łukaszewski
```

Comment:
I agree it's a nice gesture :) Thank you @TK-999 once again! :bowing_man: Wall is confusing me and something deep inside me would like my name to be removed from those files :stuck_out_tongue_closed_eyes: But what's done can't be undone, right? ;)